gsu2023.Rmd

```diff
@@ -0,0 +1,376 @@
+1. From 18th February (first assignment deadline 26th February, last day of the course in the end of May)
+1. All the material (textbook, videos, assignments, extra reading
+   material) are freely available (see below) so you can also
+   self-study at your own pace.
+1. The course is free (no cost) and possible to organize with help of volunteer TAs.
+1. This BDA course instance is aimed to support learning with peer
+   support. By following the videos and doing assignments at the same
+   time with others, you can discuss the material in assignments in
+   the course slack, there is peer-grading platform to get feedback
+   about your assignment solutions, and voluntary TAs help answering
+   questions. As everything is volunteer based we can't guarantee
+   quick responses, but at least you will get something more than when
+   studying only by yourself.
+1. This course is not the easiest Bayesian course available in
+   internet, but it can be your first Bayesian course if your
+   mathematical and programming skills are sufficient. Compared to some
+   other free online courses, this course has big emphasis in the computational
+   methods, and thus goes deeper in many parts. See the
+   prerequisites below. For easier material to start with see the end
+   of Prerequisites section below.
+1. You will not get a formal certificate for passing the course from
+   Aalto University. 
+1. The communication happens in the course slack, please don't email
+   the lecturer or TAs. The slack link has been emailed to the accepted students.
+
+All the course material is available in a [git repo](https://github.com/avehtari/BDA_course_Aalto) and via these pages for easier navigation. All the material can be used in other courses. Text and videos licensed under CC-BY-NC 4.0. Code licensed under BSD-3.
+
+## Registration
+
+You can apply for the course by filling [a registration form](https://forms.gle/azapZxfcy5sS6fhj7). The registration form asks several pre-requisites questions, and filling the form takes some time. We will email whether you have been accepted to the course at latest 17th February (Finland time). If you are not accepted for the course, you can still self-study all the material.
+
+## TA Registration
+
+We have some volunteer TAs already, but a few more would be great.
+All TAs will get a personal certificate from the lecturer Aki Vehtari
+if they actively participate helping students, answering questions,
+and possibly organize some TA sessions. We assume to have enough TAs
+that no-one needs to take part every week of the course and you can
+drop out if other obligations require so. The lecturer and two head
+TAs will support TAs.
+
+To register as volunteer TA fill in your information (email, country,
+prerequisites check, a brief comment on justification why you think
+you can be a TA)  in [TA registration form](https://forms.gle/7KQbnEC3nxb4MwYM9).
```

Comment:
In the `TA registration form` link, the last question is a "General Comment about you" and it is required. Should we make it optional or was it intentional?

Reply:
we think it would be useful for context about who the person is, more so than the other things in the form!